src/contract.js

```diff
@@ -11,13 +11,17 @@ class Contract {
 
   getDetails(hash, filterOptions) {
     if (is.notHash(hash)) return Promise.reject(new Error(NO_ADDRESS))
-    return get(this.web3data, {hash, endpoint: ENDPOINT, filterOptions})
+    return get(this.web3data, {
+      pathParam: hash,
+      endpoint: ENDPOINT,
+      filterOptions
+    })
   }
 
   getFunctions(hash, filterOptions) {
     if (is.notHash(hash)) return Promise.reject(new Error(NO_ADDRESS))
     return get(this.web3data, {
-      hash,
+      pathParam: hash,
```

Comment:
@taylorjdawson left you a comment in slack on this. Keep in mind for future refactor